src/com/yaji/Main.java

```diff
@@ -1,10 +1,12 @@
 package com.yaji;
 
+import java.text.ParseException;
+import java.util.Date;
+
 public class Main {
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws ParseException {
```

Comment:
The main method should **never** throw any errors or exceptions. Instead one should make a try-catch block around the code and handle the error in a proper way. Otherwise the program just crashes.